operator-common/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.strimzi.operator.common.Util;
+
+import java.math.BigInteger;
+import java.security.cert.CertificateEncodingException;
+import java.security.cert.X509Certificate;
+
+/**
+ * Certificate utility methods
+ */
+public class CertUtils {
```

Comment:
Any reason this is in `operator-common` and not in `cluster-operator`? Does not seem to be used anywhere else then in `cluster-operator` from a quick look.

Reply:
> Does not seem to be used anywhere else then in cluster-operator from a quick look.
Ack. Will move it.